content/en/docs/reference/controllers/daemonset.md

```diff
@@ -0,0 +1,102 @@
+---
+toc_hide: true
+title: DaemonSet controller
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+The DaemonSet {{< glossary_tooltip term_id="controller" text="controller" >}} manages
+{{< glossary_tooltip term_id="pod" text="Pods">}} configured via a
+{{< glossary_tooltip term_id="daemonset" >}} object, to make sure that some
+(or all) {{< glossary_tooltip term_id="node" text="Nodes">}} run a copy of a
+Pod.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+The DaemonSet controller is built in to the {{< glossary_tooltip term_id="kube-controller-manager" >}}.
+
+## Controller behavior
+
+The DaemonSet controller ensures that all eligible nodes run a copy of a Pod,
+by creating Pods with a Node assignment (one Pod for each eligible Node).
+
+This controller creates one Pod for each Node that the DaemonSet matches.
+It creates new Pods in batches, doubling the batch size each time a request
+succeeds. This helps to avoid sending a very large number of API requests that
+will ultimately all fail with the same error.
+
+To track changes efficiently, this controller labels new Pods using a hash
+of the the DaemonSet the Pod belongs to.
+
+If this controller is managing a Pod on a Node that no longer matches
+the DaemonSet, or where the Node no longer exists, the controller will remove
+that Pod.
+
+If a DaemonSet's Pods fail the controller recreates them, applying a rate
+limit to avoid a hot loop of killing and then recreating broken Pods.
+
+## How DaemonSet Pods are scheduled
```

Comment:
For this section maybe you'll want to take from #17476 restructuring to make sure the default behavior is clear. Or we can tag that issue and move it into here after this PR is merged, depends really how you see this restructure progressing